schema.json

```
@@ -1,329 +1,423 @@
+      "example": "PAYS-99100 pour la France entière    OU    REG-84;DEP-05 pour la région Auvergne-Rhône-Alpes et le département des Hautes-Alpes",
+      "type": "string",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "eligibilite_geographique_exclusions",
+      "description": "Couverture géographique exclues du dispositif d'aide. Champs à écrire en utilisant le formalisme du Code Officiel Géographique du COG : https://www.insee.fr/fr/information/5230987. Séparer chaque code COG par des points virgules. Cette propriété a précédence sur les \"eligibilite_geographique\" s'il y a recouvrement.",
+      "example": "REG-84;DEP-05 pour la région Auvergne-Rhône-Alpes et le département des Hautes-Alpes",
+      "type": "string"
+    },
+    {
+      "name": "eligibilite_sectorielle",
+      "description": "Couverture sectorielle du dispositif en langage naturel.",
+      "example": "Tous secteurs d’activité, sauf les industries gérées par le programme PACTE Industrie",
+      "type": "string"
+    },
+    {
+      "name": "eligibilite_sectorielle_naf",
+      "description": "Couverture sectorielle du dispositif identifiés via les codes NAF - Nomenclature d'Activités Française. Peut être n'importe niveau de code NAF (section, division, groupe etc.). Séparér chaque code NAF par des points virgules. Par défaut il est possible d'écrire A; B; C; D; E; F; G; H; I; J; K; L; M; N; O; P; Q; R; S; T; U - correspondant à toutes les sections NAF.",
+      "example": "A;55.20Z",
+      "type": "string",
+      "constraints": {
+        "required": true
+      }
+    },
+    {
+      "name": "eligibilite_sectorielle_exclusions",
+      "description": "Identifiés via leurs codes NAF. Peut être n'importe quel niveau de code NAF (section, division, groupe etc.). Séparér chaque code NAF par des points virgules. Cette propriété a précédence sur le champ précédent s'il y a recouvrement. Par exemple, si `eligibilite_sectorielle_naf` = `A` et `eligibilite_sectorielle_exclusions` = `02`, alors les secteurs éligibles s'entendent comme `Agriculture, sylviculture et pêche` hors `Division 02 : Sylviculture et exploitation forestière`.",
+      "example": "02;09.10Z",
+      "type": "string"
+    },
+    {
+      "name": "eligibilite_effectif",
+      "description": "Description des conditions d'éligibilité relatives à l'effectif en langage naturel.",
+      "example": "Exemple 1 : Moins de 50 salariés - Exemple 2 : Entre 20 et 500 salariés - Exemple 3 : Toutes tailles",
+      "type": "string"
+    },
+    {
+      "name": "eligibilite_effectif_min",
+      "description": "Nombre minimum de salarié pour bénéficier de l'aide. Laisser vide si non applicable.",
+      "example": "50",
+      "type": "number"
```

cf https://specs.frictionlessdata.io//table-schema/#integer
```suggestion
      "example": 50,
      "type": "integer"
```